Sources/NIO/AddressedEnvelope.swift

```diff
@@ -20,15 +20,35 @@
 public struct AddressedEnvelope<DataType> {
     public var remoteAddress: SocketAddress
     public var data: DataType
+    /// Any MetaData associated with this `AddressedEnvelope`
+    public var metaData : MetaData? = nil
 
     public init(remoteAddress: SocketAddress, data: DataType) {
         self.remoteAddress = remoteAddress
         self.data = data
     }
+    
+    /// Any MetaData associated with an AddressedEnvelope
+    public struct MetaData {
```

Comment:
`Metadata` is one word, and so should be spelled...well, the way I just spelled it.

Reply:
done.